pkg/kubelet/cm/dra/state/state_checkpoint.go

```diff
@@ -51,9 +51,9 @@ type ClaimInfoState struct {
 	// PodUIDs is a set of pod UIDs that reference a resource
 	PodUIDs sets.Set[string]
 
-	// CDIDevices is a list of CDI devices returned by the
+	// CDIDevices is a map of KubeletPluginName --> CDI devices returned by the
 	// GRPC API call NodePrepareResource
-	CDIDevices []string
+	CDIDevices map[string][]string
```

Comment:
in my https://github.com/kubernetes/kubernetes/pull/115847 I add the className (resourceClass). shall we move it to here or I will handle it after yours is merged?

Reply:
You added it to the `CDIDevices`? I don't quite understand.

Reply:
no,  to the  ClaimInfoState struct